util/devel/chplspell

```diff
@@ -50,37 +50,43 @@ NORMALGLOBS=("-name" "*.chpl"  "-o"
              "-name" "*.h"     "-o"
              "-name" "README*" "-o"
              "-name" "*.md"    "-o"
-             "-name" "*.rst"   "-o"
-             "-name" "*.1" )
-# *.1 are man pages
+             "-name" "*.rst" )
```

Comment:
The ``NORMALGLOBS`` array is concatenated to build a string to pass on ``find``'s commandline.  ``-o`` is ``find``'s ``or``.  So the lack of one on this line means there's not another test after it to or with.

(It's stored as an array so that bash doesn't expand the ``*``'s, but still passes the terms to ``find`` as separate tokens.)